roles/adf_app/defaults/main.yml

```diff
@@ -25,3 +25,6 @@ adf_app_default_configuration:
     redirectSilentIframeUri: "{protocol}//{hostname}{:port}/assets/silent-refresh.html"
     redirectUri: "/"
     redirectUriLogout: "/"
+  plugins:
+    processService: false
+    microsoftOnline: false
```

Comment:
this is adw specific, should be set as `adf_app_configuration` at the playbook level, otherwise it get set also for `acc` which doesn't implement any of this.